components/search/stories/search.stories.js

```diff
@@ -16,29 +19,37 @@ export default {
 			options: ["s", "m", "l", "xl"],
 			control: "select",
 		},
-		isQuiet: {
-			name: "Quiet styling",
+		isDisabled: {
+			name: "Disabled",
 			type: { name: "boolean" },
 			table: {
 				type: { summary: "boolean" },
-				category: "Component",
+				category: "State",
 			},
 			control: "boolean",
 		},
-		isDisabled: {
-			name: "Disabled",
+		isFocused: {
+			table: { disable: true },
+			type: { name: "boolean" },
+		},
```

Comment:
I added this because TextField uses this class in it's Storybook, so we can use this arg to [pass through to TextField](https://github.com/adobe/spectrum-css/pull/2673/files#diff-5a941796927ca4c2dd96d02d4b51ac05e7f1a0c2d9a7ba3292cf2948a67245baR33) to show the focus state in our Chromatic testing preview for Search.